debian/control

```diff
@@ -93,6 +93,7 @@ Build-Depends: automake,
                tox <pkg.ceph.check>,
                python3-coverage <pkg.ceph.check>,
                python3-dateutil <pkg.ceph.check>,
+               python3-pkg-resources <pkg.ceph.check>,
```

Comment:
This conflict (vs `debian/ceph-mgr-modules-core.requires` file in main) needs to be documented in the commit message, in particular explaining the reason why this dependency is a `pkg.ceph.check` dependency instead of regular one.

Reply:
Can you help me understand why is it a conflict?

Reply:
Well, it's not a straight cherry-pick, is it?  The original commit in main adds a line to `debian/ceph-mgr-modules-core.requires` file, here you are adding a _different_ line to `debian/control` file.

Reply:
Oh, the original commit couldn't be cherry-picked as is since we introduced some changes in the organization of
the `debian` dir (`ceph-mgr-modules-core.requires` doesn't exist yet). 
All the packages listed in main's `ceph-mgr-modules-core.requires` are marked as `pkg.ceph.check` in P/Q `debian/control`.
The changes here are the same as in Pacific's backport #49695.
I can add this explanation to the commit's message if this was your intent.

Reply:
> I can add this explanation to the commit's message if this was your intent.

Yes -- this should be done as rule, not upon request.

Reply:
Added, thank you.

Reply:
> All the packages listed in main's `ceph-mgr-modules-core.requires` are marked as `pkg.ceph.check` in P/Q `debian/control`.

This statement appears to be false for 2 out of 6 packages:
```
  88                python3-cherrypy3,
  89                python3-natsort,
  91                python3-pecan <pkg.ceph.check>,
  95                python3-dateutil <pkg.ceph.check>,
  98                python3-requests <pkg.ceph.check>,
 103                python3-werkzeug <pkg.ceph.check>,
```
Am I looking in the wrong place?

>	Therefore, python3-pkg-resources is listed as <pkg.ceph.check> in debian/control

All of this begs the question: are these backports adding this line to the right section to begin with?  I think we need to clarify whether `python3-pkg-resources` package is:

1. a general build dependency for Ceph
2. a build dependency for Ceph that is needed only for `make check`
3. a runtime dependency for `ceph-mgr-modules-core` package
4. some combination of 1-3?

> The changes here are the same as in Pacific's backport #49695.

Adding @ljflores who approved #49695.